frameworks/elastic/universe/config.json

```diff
@@ -355,13 +360,13 @@
           "description": "By default all script contexts (search,update,aggs,plugin) are allowed to be executed. This can be modified using the setting script_allowed_contexts. Only the contexts specified as part of the setting will be allowed to be executed. To specify no contexts are allowed, set script_allowed_contexts to be none. Empty string \"\" means all contexts (default).",
           "type": "string",
           "default": ""
-        }, 
+        },
         "repositories_url_allowed_urls": {
           "description": "URL Repository supports the following protocols: \"http\", \"https\", \"ftp\", \"file\" and \"jar\". URL repositories with http:, https:, and ftp: URLs have to be whitelisted by specifying allowed URLs in the repositories_url_allowed_urls setting. This setting supports wildcards in the place of host, path, query, and fragment.",
           "type": "string",
           "default": ""
         },
-       "network_tcp_no_delay": {
+        "network_tcp_no_delay": {
```

Comment:
Seems like an extra space.

Reply:
The space added fixes the indentation, no?

Reply:
Good point!